src/python/test/test_dataset.py

```diff
@@ -16,20 +16,23 @@ def setUp(self):
         self.base_dtypes = [
             'int8', 'int16', 'int32', 'int64',
             'uint8', 'uint16', 'uint32', 'uint64',
-            'float32', 'float64'
+            'float32', 'float64',  # 'float128',
+            'complex64', 'complex128',  # 'complex256',
```

Comment:
I've observed good behaviour with complex64 and complex128 in both Zarr and N5 format, but complex256 is problematic, possibly due to a dependency's limited support for float128. I don't particularly need that precision level, but it is the last precision level supported by npy and could be a good addition in the future once the problem is better understood.

Happy to remove all mentions of float128 and complex256, or leave them as is to raise visibility about missing functionality.

Reply:
I think leaving it as a comment is good to point to the possibility of adding it.